src/input.service.ts

```diff
@@ -268,7 +267,10 @@ export class InputService {
    * Inputpane and boolean to indicate whether it's visible
    */
   private inputPane = (<any>window).Windows ? Windows.UI.ViewManagement.InputPane.getForCurrentView() : null;
-  public keyboardVisible = new BehaviorSubject(false);
+
+  public get keyboardVisible(): boolean {
+    return (this.inputPane.occludedRect.y !== 0);
```

Comment:
Unnecessary parens, also you probably want `this.inputPane && this.inputPane.occludedRect.y !== 0`